src/test/java/fr/minuskube/inv/ClickableItemTest.java

```diff
@@ -1,24 +1,22 @@
 package fr.minuskube.inv;
 
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import java.util.concurrent.atomic.AtomicBoolean;
 import org.bukkit.Material;
 import org.bukkit.event.inventory.InventoryClickEvent;
 import org.bukkit.event.player.PlayerInteractEvent;
 import org.bukkit.inventory.ItemStack;
 import org.junit.Test;
 
-import java.util.concurrent.atomic.AtomicBoolean;
-
-import static org.junit.Assert.*;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.when;
-
 public class ClickableItemTest {
 
     @Test
     public void testNone() {
         ClickableItem item = ClickableItem.NONE;
 
-        assertNull("The item from ClickableItem.NONE is not null", item.getItem());
+        assertNotNull("The item from ClickableItem.NONE is not null", item.getItem());
```

Comment:
If you use your change, then the error message is wrong and you need to check for an ItemStack with the `Material.AIR` or revert this with the ClickableItem change regarding `NONE`